doc/user/data/sql_funcs.yml

```diff
@@ -23,17 +23,6 @@
   - signature: 'count(x: T) -> int'
     description: Number of non-_NULL_ inputs.
 
-  - signature: jsonb_agg(expression) -> jsonb
-    description: Aggregate values (including nulls) as a jsonb array.
-    url: jsonb_agg
-
-  - signature: jsonb_object_agg(keys, values) -> jsonb
-    description: Aggregate keys and values (including nulls) as a jsonb object.
-    url: jsonb_object_agg
-
-  - signature: 'array_agg(x: T) -> T[]'
```

Comment:
These were previously alphabetic, except for `array_agg` which is maybe the only one that should get moved?